extensions/Khronos/KHR_draco_mesh_compression/README.md

```diff
@@ -0,0 +1,171 @@
+
+```javascript
+
+"mesh" : {
+    "primitives" : [
+        {
+            "attributes" : {
+                "POSITION" : 11,
+                "NORMAL" : 12,
+                "TEXCOORD_0" : 13,
+            },
+            "indices" : 10,
+            "mode" : 4
+            "extensions" : {
+                "KHR_draco_mesh_compression" : {
+                    "bufferView" : 5,
+                    "indicesCount" : 1000,
+                    "vertexCount" : 500,
+                    "attributes" : [
+                        {
+                            "semantic" : "POSITION",
+                            "componentType" : 5126,
+                            "type" : "VEC3",
+                        },
+                        {
+                            "semantic" : "NORMAL",
+                            "componentType" : 5126,
+                            "type" : "VEC3",
+                        },
+                        {
+                            "semantic" : "TEXCOORD_0",
+                            "componentType" : 5126,
+                            "type" : "VEC2",
+                        },
+                    ]
+                }
+            }
+        },
+    ]
+}
+
+"bufferViews" : [
+    // ...
+    // bufferView of Id 5
+    {
+        "buffer" : 10,
+        "byteOffset" : 1024,
+        "byteLength" : 10000
+    }
+    // ...
+}
+
+```
+We will explain each of the property in the following sections.
+#### bufferView
+The `bufferView` property points to the buffer containing compressed data. The data should be passed to a mesh decoder and decompressed to a
+mesh.
+
+#### indicesCount, vertexCount
+`indicesCount` and `vertexCount` are references for verifying the decompression of
+mesh.
+
+#### attributes
+The decompressed mesh needs to be write to the memory for uploading to GPU,
+including indices and attributes data. `attributes` is used to define the
+order of the attributes. For example, if we have the above `attributes`, then we need to write the attributes to vertex buffer like
+
+     --------------------------------------------------
+    |    POSITION    |    NORMAL    |    TEXCOORD_0    |  
+     --------------------------------------------------
+
+`componentType` and `type` are duplicated properties in
+`accessor` of `attributes`. The purpose of having these in the extension is to
+define how to write the decompressed mesh to buffer for uploading to GPU without
+refering to the sibling `attributes` node.
+
+The extension currently don't support morph targets, e.g. `targets` is used in
+`primitive`. 
+
+#### Restrictions on geometry type
+When using this extension, the `mode` of `primitive` could only be one of
+`POINTS`, `TRIANGLES` and `TRIANGLE_STRIP` and the mesh data will be decoded accordingly. For example, if `mode` is `POINTS`, then the
+decompressed geometry will be a point cloud.
+
+### JSON Schema
+
+For full details on the `KHR_draco_mesh_compression` extension properties, see the schema:
```

Comment:
It could be cool to try the latest wetzel (https://github.com/AnalyticalGraphicsInc/wetzel/pull/4) to generate reference doc to paste in here like the glTF spec has.